src/generators/operationGenerator.ts

```diff
@@ -216,11 +217,12 @@ function buildResponses({ responses }: OperationSpecDetails): string[] {
   let parsedResponses: string[] = [];
   responseCodes.forEach(code => {
     // Check whether we have an actual mapper or a string reference
-    const { bodyMapper, headersMapper } = responses[code];
+    const { bodyMapper, headersMapper, isError } = responses[code];
     const bodyMapperString = buildMapper(bodyMapper, "bodyMapper");
     const headersMapperString = buildMapper(headersMapper, "headersMapper");
+    const isMapperString = isError ? `isError: ${isError}` : "";
```

Comment:
What do you think about renaming this to `isError` or `isErrorMapperString`?

Reply:
changed it to isErrorMapperString